packages/engine/test/testContext.js

```diff
@@ -50,7 +50,7 @@ const testContext = async ({ lowdefy, operators, rootBlock, initState = {} }) =>
   _internal.Actions = new Actions(ctx);
   _internal.Requests = new Requests(ctx);
   _internal.RootBlocks = new Blocks({
-    areas: { root: { blocks: [_internal.rootBlock] } },
```

Comment:
Why did you do this, because we still do:
```js
  _internal.RootBlocks = new Blocks({
    areas: { root: { blocks: [_internal.rootBlock] } },
    context: ctx,
  });
```
in `createContext`